modules/recording/recording/dashboard.tsx

```diff
@@ -37,22 +38,20 @@ const Dashboard = () => {
         await audioRecorderRef.current.startRecording(microphone);
         messageHandler.info(t("recording.started"));
         setRecordingState("recording");
+        startVisualization();
       } catch (error) {
         messageHandler.handleError((error as Error).message);
       }
-      return
-    }
-    if (recordingState === "recording") {
+    } else if (recordingState === "recording") {
```

Comment:
I remove the `else` and you put then back.

Don't add `else` they are a smell code. If you feel that you really need then call me.

Use `if-return`